cinema/models.py

```diff
@@ -5,6 +5,33 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    actors = models.ManyToManyField("Actor", related_name="movies")
+    genres = models.ManyToManyField("Genre", default=[], related_name="movies")
```

Comment:
The `default` parameter is not valid for `ManyToManyField`. You should remove `default=[]` from this field definition. `ManyToManyField` does not support a default value because it represents a relationship rather than a single value.